docs/README.md

```diff
@@ -17,12 +17,12 @@
   under the License.
 -->
 
-# RaySQL Design Documentation
+# Datafusion Ray Design Documentation
```

Comment:
nit: can we capitalize it as `DataFusion` rather than `Datafusion`.

```suggestion
# DataFusion Ray Design Documentation
```

Reply:
done. thanks for reminding